### DISCUSSION TITLE
My first new story

### DISCUSSION THREAD

**Discussion by frigus02** [frigus02/wfps2023-starter#2] [General]
Hi 👋

**Comment by frigus02** [comment]
A comment

**Reply by frigus02** [reply]
A reply

**Reply by hybrist** [reply]
Reply to reply to comment.

P.S.: And it was edited later.

P.P.S.: It was even edited twice.

**Comment by frigus02** [comment]
Another comment

**Reply by frigus02** [reply]
Reply to another comment

**Comment by hybrist** [comment]
A _comment_ that contains **markdown**. See also: https://github.com/frigus02/wpfs2023-starter/discussions/2#discussioncomment-6382464.

![fun](https://media.giphy.com/media/LujfegRWzJn3y/giphy.gif)

**Comment by frigus02** [comment]
hello world from graphql explorer

**Comment by frigus02** [comment]
another comment from graphql explorer
